apps/web/pages/tasks.tsx

```diff
@@ -34,12 +34,19 @@ export default function Tasks(props) {
       }
 
       <AddTaskItem addTaskItem={addTaskItem} isLoading={isLoading} />
+      <Flex
+      overflowY="scroll"
+      sx = {{
+      '::-webkit-scrollbar' :{display: 'none'}
+      }}
+      >
       <VStack
+        h="420px"
         alignItems="flex-start"
         width={"600px"}>
         {
           props?.tasks?.data?.length > 0 ?
-            props?.tasks?.data?.map(task => <TaskItem task={task} key={task.id} />) : (
+            props?.tasks?.data?.map(task => <TaskItem  task={task} key={task.id} />) : (
               <>
                 <Flex alignItems="center" gap={8} marginTop="8" p="4" justifyContent={"center"}>
                   <img size='sm' alt="empty" src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTi8Y6jlyyGKX4Xok7Q6ro0TwI-hCCHLP1zovBevgm_JsTiTnbQXbT9UMCt2YOhDBOjHwo&usqp=CAU' height="300px" />
```

Comment:
Try using the [Image](https://nextjs.org/docs/api-reference/next/image) component provided by `Next.js`.

Reply:
![Screenshot 2022-07-08 161741](https://user-images.githubusercontent.com/71399438/178001359-e8f88ada-f49b-489b-a14f-a5e7c7f0bddc.png)
![Screenshot 2022-07-08 161843](https://user-images.githubusercontent.com/71399438/178001392-4602f513-daac-4f81-b963-27e653d56e5e.png)